src/server/plugins/reactHtml/index.jsx

```diff
@@ -268,6 +275,7 @@ export function getBody({
         window.__render_mode__ = '${renderMode}';
         window.__HOLOCRON_EXTERNALS__ = ${jsonStringifyForScript(getRequiredExternalsRegistry())};
       </script>
+      ${renderUseNativeIntlPolyfill(nonce)}
       ${assets}
       ${renderI18nScript(clientInitialState, bundlePrefixForBrowser)}
```

Comment:
If using native Intl, this file should not be included

Reply:
As discussed, I have added a guard clause to this function to return early if the env var equals 'true' in [edb789a](https://github.com/americanexpress/one-app/pull/1258/commits/edb789aa4e5e79e34d18e794e45fc2c3de99841f)